catalog/core/catalog-core-standardframework/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -309,6 +309,8 @@
         <property name="productCacheDirectory" value=""/>
         <property name="context" ref="blueprintBundleContext"/>
         <property name="xmlConfigFilename" value="reliableResource-hazelcast.xml"/>
+        <!--            TODO: Temporary injection until removal            -->
```

Comment:
@shaundmorris do you mean the cache, or the manager? As of #865, `CatalogFrameworkImpl` has no knowledge of the existence of the `ResourceCache` but still needs to access the `ReliableResourceDownloadManager`. Though I'm not sure that the `ResourceCacheImpl` should be getting a reference to the `ReliableResourceDownloadManager` which is what it looks like is happening here.